examples/surface_mesh_cylinder_from_3rd_party_geometry_file.py

```diff
@@ -0,0 +1,19 @@
+import flow360 as fl
+from flow360.component.geometry import Geometry
+from flow360.examples import CylinderGeometry
+
+fl.Env.dev.active()
+
+geometry = Geometry.from_file(
+    CylinderGeometry.geometry, name="testing-cylinder-geometry", solver_version="release-24.2.2"
+)
+geometry = geometry.submit()
+
+params = fl.SurfaceMeshingParams(CylinderGeometry.surface_json)
+surface_mesh = fl.SurfaceMesh.create(
+    geometry_id=geometry.id, params=params, name="cylinder-geometry-new-python-client"
```

Comment:
maybe we can just have name: cylinder-surface-mesh?

Reply:
fixed